make.inc.macosx_gcc-10

```diff
@@ -0,0 +1,54 @@
+# Makefile variable overrides for Mac OSX compilation with GCC v.10.*
+#
+# Use this if you'll need to link against gfortran.
+#
+# Copy this file to make.inc, and if needed edit for your setup.
+#
+# Don't forget also to adjust g++ to g++-10 at start of setup.py
+#
+# Barnett 10/27/18. Input from Yu-Hsuan Shih, Amit Moskovich.
+# modified 12/06/2020
+
+# By default we use clang/LLVM (which is aliased to /usr/lib/gcc, etc).
+# This make.inc is if you want to override this.
+# Get gcc from brew then use, eg:
+CXX=g++-10
+CC=gcc-10
+FC=gfortran
+# For Catalina, you may need to edit the above to g++-10, etc.
```

Comment:
this comment is now irrelevant, right?

Reply:
Yes, I think we do not need this comment. For any MacOS on ci, one has to use g++-10 or gcc-10 with gcc installed using brew. g++ or gcc refers to clang compiler by default.